src/electron/add_tap_device.bat

```diff
@@ -59,34 +42,26 @@ if %errorlevel% neq 0 (
 
 :: Find the name of the most recently installed TAP device in the registry and rename it.
 echo Searching for new TAP network device name...
-set TAP_NAME_FILE=%tmp%\outlineinstaller-tap-device-name.txt
-find_tap_name.exe --component-id %DEVICE_HWID% --ignored-names "%NETWORK_INTERFACES_FILE%" > %TAP_NAME_FILE%
+call find_tap_device_name.bat TAP_NAME
 if %errorlevel% neq 0 (
   echo Could not find TAP device name. >&2
   exit /b 1
 )
-set /p TAP_NAME=<%TAP_NAME_FILE%
 echo Found TAP device name: "%TAP_NAME%"
+
+:: We've occasionally seen delays before netsh will "see" the new device, at least for
+:: purposes of configuring IP and DNS ("netsh interface show interface name=xxx" does not
+:: seem to be affected). Attempt to rename even if waiting timed out.
+call :wait_for_device "%TAP_NAME%"
+
```

Comment:
I think the "Attempt to rename even if waiting timed out." comment should go here instead of above, and similar after the next call to `:wait_for_device`.

Reply:
Done.